extensions/kraken/update-products.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+var KrakenClient = require('kraken-api');
+var kraken = new KrakenClient();
+
+var mapping;
+var products = [];
+
+function addProduct(base, quote, altname) {
+    products.push({
+        asset: base,
```

Comment:
not sure if necessary, but I noticed it in the GDAX `products`:
https://github.com/carlos8f/zenbot/pull/205/files#diff-4899b562666048f7641da71b0434f132R15

might need to add `id`?